app/dashboard/static/js/app/view-socs-soc-job-kernel.2017.4.js

```diff
@@ -701,10 +533,15 @@ define([
             });
     });
 
-    gSessionStorage = storage('soc-' + gSoc + '-' + gJob + '-' + gKernel);
-    gResultFilter = filter('data-filter');
-    getBoots();
-    registerEvents();
+
+    gPlansTable = table({
+        tableId: 'planstable',
+        tableLoadingDivId: 'table-loading',
+        tableDivId: 'table-div',
+    });
+
+    setTimeout(getPlans, 10);
+    setTimeout(getTestCount, 10);
 
     init.hotkeys();
     init.tooltip();
```

Comment:
And, how about `setTimeout` for these functions?  They have timeouts in other views.

Reply:
good point, updated.